test/test_helper.rb

```diff
@@ -55,3 +55,5 @@
 
 # Prevent this deprecation warning from breaking the tests.
 Rake::FileList.send(:remove_method, :import)
+
+ActiveSupport::TestCase.test_order = :sorted
```

Comment:
It looks like this was not supported back in test/unit 3.1.5 which ActiveRecord 3.1 used. Would you mind adding a version flag around this for AR 4.x or greater?